src/box/iproto.cc

```diff
@@ -2352,14 +2387,21 @@ struct iproto_cfg_msg: public cbus_call_msg
 };
 
 static inline void
-iproto_thread_fill_binary(struct iproto_thread *iproto_thread,
```

Comment:
Nit: you can leave this function as is and call it in a loop inside fill_binaries()

Reply:
I had to fix the function a little, but I kept it